attest/eventlog.go

```diff
@@ -506,7 +506,7 @@ func ParseEventLog(measurementLog []byte) (*EventLog, error) {
 	if err != nil {
 		return nil, fmt.Errorf("parse first event: %v", err)
 	}
-	if e.typ == eventTypeNoAction {
+	if e.typ == eventTypeNoAction && len(e.data) >= binary.Size(specIDEventHeader{}) {
```

Comment:
Can you please add a comment here? Without a unit test, if we ever refactor this it'd be helpful to have at least some understanding of why this is needed :)

Reply:
Added an unit-test and a comment. The comment I placed into the unit-test.

And do not hesitate to to be brutal on review :)